app/pricecheck.js

```diff
@@ -1,37 +1,50 @@
 var itemName = document.title.split("Listings for ")[1];
-
 var LoungeUser = new User();
+
 chrome.storage.local.get(['currencyConversionRates'], function(result) {
+
+    var GetCSGLSchemaUrl = 'http://csgolounge.com/api/schema.php';
+    var CheckCSGLHtml = '<span class="btn_small btn_grey_white_innerfade" id="csglpricecheck"><span>Check CSGOLounge.com item value</span></span>';
+
     currencies = result.currencyConversionRates || {};
-    LoungeUser.loadUserSettings(function() {
+    LoungeUser.loadUserSettings(renderAndBindGetPrice);
+
+    function renderAndBindGetPrice(){
```

Comment:
I still don't get what's the purpose of wrapping every small thing into its own function. Please explain. pricecheck.js does only one thing and is only CS:GO specific.